xero_app.rb

```diff
@@ -13,14 +13,21 @@
 use Rack::Session::Pool
 set :haml, :format => :html5
 
-# We initialise an instance of the Xero API Client here so we can
-# make calls to the API later.
-@@xero_client = XeroRuby::ApiClient.new(credentials: {
+# Setup the credentials we use to connect to the XeroAPI
+CREDENTIALS = {
   client_id: ENV['CLIENT_ID'],
   client_secret: ENV['CLIENT_SECRET'],
   redirect_uri: ENV['REDIRECT_URI'],
   scopes: ENV['SCOPES']
-})
+}
+
```

Comment:
@jkueh going to merge but let me know if this looks good.

![you da man](https://media1.giphy.com/media/QyW2vfX8racdS1k4st/giphy.gif?cid=ecf05e478jbbgxztps5gkjkcph04zs6yzhj05fyri1eqt5lm&rid=giphy.gif)